README.md

```diff
@@ -103,6 +103,22 @@ let server = new FastBootAppServer({
 server.start();
 ```
 
+## Pre and Post FastBoot middleware hooks
+
+If you need something less than a custom server and just want to run some middleware
+before or after FastBoot runs, the server provides hooks for you to do so:
+
+```js
+// Custom Middlewares
+function modifyRequest(req, res, next) { /* do pre-fastboot stuff to `req` */ };
+function handleErrors(err, req, res, next) { /* do error recovery stuff */ };
+
+const server = FastBootAppServer({
+  preFastbootMiddlewares: function (app) { app.use(modifyRequest); },
+  postFastbootMiddlewares: function (app) { app.use(handleErrors); }
```

Comment:
This is a little verbose, and I think `middleware` is an irregular plural, right? "Middlewares" sounds strange to me.

Maybe something like:

* `beforeMiddleware`
* `afterMiddleware`
?